multidict/__init__.py

```diff
@@ -57,3 +58,15 @@
 
 
 upstr = istr
+
+# Inspired by Numpy
+
+
+def get_include() -> str:
+    """Get multidict headers for compiling multidict with other
+    C Extensions or cython code"""
+    # NOTE: Import pathlib later so that were not being slow during
+    # other use-cases
+    import pathlib
+
+    return str(pathlib.Path(__file__).parent)
```

Comment:
Honestly, I'd avoid string-based path APIs. In Python, it's best to always use pathlib objects up until the point where strings are needed to interface with external systems. I think it's best to just return a pathlib object and let the caller convert it however they prefer. Maybe also `.resolve()` it before returning for a good measure.

Reply:
I think it's not obvious to me (or probably @webknjaz) what the purpose of this function is? Is this some sort of standard interface to something?

It's not called anywhere in this PR and it's being exported as a public function. The docstring says it gets headers, but it appears to just return the path that multidict is installed in. Do people need this function, when they can just do `multidict.__path__`?